core/src/storage.rs

```diff
@@ -68,10 +80,16 @@ impl LocalStorage {
         Ok(())
     }
 
-    pub fn get_available_releases(&self) -> Result<Vec<Release>> {
-        Ok(fs::read_dir(&self.data_dir)?
+    pub fn get_available_releases(&self, game: Game) -> Result<Vec<Release>> {
+        let game_path = self.game_path(game);
+
+        if !game_path.exists() {
+            return Ok(Vec::default());
```

Comment:
`Vec::new()` or `vec![]` is more explanatory than `Vec::default()`. Let's change this to one of them.